src/App.css

```diff
@@ -59,7 +59,6 @@ picture {
   display: block;
 }
 
-button,
```

Comment:
@jssckbl  I don't know if you can see, but when you move the mouse over `map` or `list` button, the icon change color. I couldn't see it again. 
![image](https://user-images.githubusercontent.com/8689897/131945075-7d9729a3-a89c-4da3-90ee-5c1d190d2a43.png)

Reply:
I noticed that as well. Still trying to figure it out 😬 

If the user is clicking between `Map` and `List` then the icons remain. 

Once a user clicks on another button other than one of those two, then the icon gets weird again.

Reply:
I'm not sure how to resolve it yet. But working on it 👍

Reply:
Thank you for your help in resolving this, @alxmcr 🥳